slither/tools/read_storage/__main__.py

```diff
@@ -104,11 +107,62 @@ def parse_args() -> argparse.Namespace:
         default="latest",
     )
 
+    parser.add_argument(
+        "--diff-block",
+        help="Calculates storage value differences between block numbers specified as --block and --diff-block arguments. Implies --value and --block (if not set, defaults to latest).",
+        default=None,
+    )
+
     cryticparser.init(parser)
 
     return parser.parse_args()
 
 
+def print_diff_table(orig_block: int, diff_block: int, orig_info: SlotInfo, diff_info: SlotInfo):
+
+    if orig_block == "latest" or orig_block > diff_block:
```

Comment:
It seems like this function shouldn't sort its arguments (then its misleading that one arg is called `orig_block`).